tests/GenerateTest.php

```diff
@@ -6,7 +6,7 @@ class GenerateTest extends \PHPUnit_Framework_TestCase
 {
     private function generateLocaleFilesFrom(array $arr)
     {
-        $root =  sys_get_temp_dir() . '/' . sha1(microtime(true) . mt_rand());        
+        $root =  __DIR__ . '/tmp/' . sha1(microtime(true) . mt_rand());        
```

Comment:
Is this change required?

It may be troublesome since it would require the user to have write access to this directory.
sys_get_temp_dir() would give you a directory that is safe to write to.

Reply:
Nah, it just my case at for #36.
My bad, i forgot to change it back.